test/hotspot/jtreg/runtime/CommandLine/VMDeprecatedOptions.java

```diff
@@ -55,9 +55,11 @@ public class VMDeprecatedOptions {
         // optionally added.
         ArrayList<String[]> deprecated = new ArrayList(
           Arrays.asList(new String[][] {
+            // { <flag name> , <expected default value> }
             // deprecated non-alias flags:
             {"AllowRedefinitionToAddDeleteMethods", "true"},
             {"LockingMode", "1"},
+            {"UseCompressedClassPointers", "false"},
```

Comment:
If  you run this test on an arm32, will it fail because UseCompressedClassPointers is unrecognized there?  Does this need to be conditional on 64 bit platforms?

Reply:
You are right, I fixed the test. I had tested the JVM on arm32 manually, but did not run this test.